tests/conftest.py

```diff
@@ -10,10 +10,20 @@ def __init__(self, response_dict, status_code, headers=None):
         self.status_code = status_code
         self.headers = {} if headers is None else headers
 
+        if "content-type" not in self.headers:
+            self.headers["content-type"] = "application/json"
+
     def json(self):
         return self.response_dict
 
 
+class MockRawResponse(object):
+    def __init__(self, content, status_code, headers=None):
+        self.content = content
+        self.status_code = status_code
+        self.headers = {} if headers is None else headers
```

Comment:
Should we move this `{}` default up to the parameter declaration in these mock helpers?

```python
class MockRawResponse(object):
    def __init__(self, content, status_code, headers={}):
        self.content = content
        self.status_code = status_code
        self.headers = headers
```

Not sure if there are any places in the tests that explicitly pass `headers=None` though.

Reply:
Yea, that seems like a good simplification.

I can open up a follow-up PR.